zeebe/logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LogStreamImpl.java

```diff
@@ -68,7 +70,6 @@ public void close() {
     LOG.debug("Closing {} with {} readers", logName, readers.size());
     readers.forEach(LogStreamReader::close);
     logStorage.removeCommitListener(this);
-    logStreamMetrics.remove();
```

Comment:
:exclamation: This is taken care of by closing the meter registry during a transition.